.github/workflows/python-package.yml

```diff
@@ -21,7 +21,9 @@ jobs:
           python-version: ${{ matrix.python-version }}
           cache: "poetry"
       - run: poetry install
-      - run: poetry run pytest --cloud-api-key ${{ secrets.CLOUD_API_KEY }}
+      - run: poetry run pytest
+      - run: poetry run pytest -m cloud --cloud-api-key "${{ secrets.CLOUD_API_KEY }}"
+      - run: poetry run pytest -m enterprise --enterprise-api-key "${{ secrets.ENTERPRISE_API_KEY }}"
```

Comment:
Any specific reason not to run all of the tests in one command instead of splitting it into 3? My understanding is if you pass both cloud and enterprise arguments, it'll be able to run all of the tests.